CS3230Project/View/SearchPatient.cs

```diff
@@ -0,0 +1,115 @@
+﻿using CS3230Project.Model.Accounts;
+using System;
```

Comment:
Not holding up Approval. At some point we might want to make the search by name checked on by default. Would like your opinion on this. Again this is just a comment and not holding up the Approval.

Reply:
Agreed! It gets annoying while testing. It has been fixed.